### ISSUE TITLE
Copying cell content copies next cell to the right

### ISSUE THREAD

**Issue by emschwartz** [supabase/grid#63]
# Bug report

## Describe the bug

When I click to Copy Cell Content, it copies the data from the cell to the right.

## To Reproduce

Steps to reproduce the behavior, please provide code snippets or a repository:

1. Go to the Table view
2. Right click on a cell and copy its data
3. Paste it somewhere else to see what was copied

## Screenshots

![image](https://user-images.githubusercontent.com/3262610/124339587-88f3b500-db7d-11eb-8677-9b57a999cc24.png)

Copying the cell on the left copies the data from the cell on the right to the clipboard.

**Comment by phamhieu** [comment]
Hi @emschwartz can you try to resize the column or reorder columns... so that the tanle localstorage data refreshs. Then you can try to copy cell again.

I cant replicated the issue on my side. But i guess its a localstorage problem.

**Comment by emschwartz** [comment]
I tried resizing and reordering the columns and see the same behavior. I also deleted everything from local storage and tried it again with the same results.

**Comment by phamhieu** [comment]
@emschwartz as i can't replicated this on my side, can you provide a step by step to reproduce the issue? Thank you.

**Comment by emschwartz** [comment]
Hi @phamhieu,

I just tried creating a new project and table, as well as a new table within my existing project, and was also not able to reproduce the behavior. I checked my other tables again and found that it only affects a single table -- and even within that table the copy behavior is inconsistent; sometimes it copies the correct cell and sometimes not.

I'm not really sure what's going on with this one table so I'll close this for now (if anyone else sees the same issue it may be worth investigating further).
